core/scripts/go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/smartcontractkit/chainlink-automation v0.8.1
 	github.com/smartcontractkit/chainlink-common v0.3.1-0.20241101093830-33711d0c3de7
 	github.com/smartcontractkit/chainlink/deployment v0.0.0-00010101000000-000000000000
-	github.com/smartcontractkit/chainlink/v2 v2.0.0-00010101000000-000000000000
+	github.com/smartcontractkit/chainlink/v2 v2.14.0-mercury-20240807.0.20241106193309-5560cd76211a
```

Comment:
Sidebar: why is our pseudo version 2.14 based? We just tagged through 2.18 the other day.

Reply:
cc @chainchad

Reply:
i've been wondering this too but haven't bubbled it up. thanks for calling it out